packages/dataprotector-consumer/.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Any reason why this is so low? Because of borsh?

Reply:
the nodejs TEE runtime provided by Scone is currently node 14, so dapps are very likely to use node 14 for now